department-of-veterans-affairs/caseflow
# Create Associated Prior Mail: Can view all associated prior mail

Resolves [Jira Issue Title](https://jira.devops.va.gov/browse/JIRA-42523)

# Description
In the Associated Prior Mail tab on the Correspondence Details page a table of all other correspondences are shown.

## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
1. Navigate to correspondence queue (Your Correspondences or Correspondence Cases page)
2. Select a correspondence and notice link (i.e. http://localhost:3000/queue/correspondence/bde5e263-d85b-4e8a-a49d-4a158087fc82/review_package)
3. Remove the review_package (or any other after-pathways) to navigate to the correspondence details page
    1. i.e. **_http://localhost:3000/queue/correspondence/bde5e263-d85b-4e8a-a49d-4a158087fc82_**
4. Check that details page has "Associated prior mail".
5. Click on Associated prior mail then Prior mail table with the values for Package Document Type, VA DOR, Notes, Correspondence Type are displayed.


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1544" alt="Screen Shot 2024-08-21 at 3 35 29 PM" src="https://github.com/user-attachments/assets/fc3fef7a-0b9c-496e-b6d2-0a37c777337a">


## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
